sdk/core/core-amqp/src/retry.ts

```diff
@@ -60,10 +60,10 @@ export interface RetryConfig<T> {
    */
   operationType: RetryOperationType;
   /**
-   * @property {number} [times] Number of times the operation needs to be retried in case
+   * @property {number} [maxRetries] Number of times the operation needs to be retried in case
    * of error. Default: 3.
    */
-  times?: number;
+  maxRetries?: number;
```

Comment:
Since the documentation says -  `Number of times the operation needs to be retried in case of error. Default: 3.` and not number of times 'tried', this seems correct to me.

Reply:
So, maxRetries does not include the initial attempt, please see https://github.com/Azure/azure-sdk-for-js/issues/4038#issuecomment-507890491 @ShivangiReja